src/components/form-editor/form-editor.component.tsx

```diff
@@ -54,13 +54,22 @@ const FormEditor: React.FC = () => {
     formUuid && (isLoadingClobdata || isLoadingForm);
 
   useEffect(() => {
-    if (formUuid && clobdata && Object.keys(clobdata).length > 0) {
-      setSchema(clobdata);
-      localStorage.setItem("formJSON", JSON.stringify(clobdata));
-    } else if (formUuid && !isLoadingFormOrSchema && !schema) {
-      setShowLoadDraftSchemaModal(true);
+    if (formUuid) {
+      if (
+        !isLoadingFormOrSchema &&
+        clobdata &&
+        Object.keys(clobdata).length > 0
+      ) {
+        setSchema(clobdata);
+        localStorage.setItem("formJSON", JSON.stringify(clobdata));
```

Comment:
This reminds me: we should probably have some naming conventions for localStorage items, especially given this is a MF project, e.g., `@openmrs/esm-form-builder:formJson` or something. (Don't need to do this now, just flagging the discussion point)